pkg/csi/service/common/common_controller_helper.go

```diff
@@ -227,7 +227,8 @@ func UseVslmAPIs(ctx context.Context, aboutInfo vim25types.AboutInfo) (bool, err
 // ValidateControllerExpandVolumeRequest is the helper function to validate
 // ControllerExpandVolumeRequest for all block controllers.
 // Function returns error if validation fails otherwise returns nil.
-func ValidateControllerExpandVolumeRequest(ctx context.Context, req *csi.ControllerExpandVolumeRequest) error {
+func ValidateControllerExpandVolumeRequest(ctx context.Context, req *csi.ControllerExpandVolumeRequest,
```

Comment:
add another func in the wcp/controller_helper.go for specific validation for block volume with RWX.